platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapView.java

```diff
@@ -535,6 +536,35 @@ public void setStyleUrl(@NonNull String url) {
     nativeMapView.setStyleUrl(url);
   }
 
+  /**
+   * Loads a new style from the specified offline region definition and moves the map camera to that region.
+   *
+   * @param definition the offline region definition
+   * @see OfflineRegionDefinition
+   */
+  public void setOfflineRegionDefinition(OfflineRegionDefinition definition) {
+    if (destroyed) {
+      return;
+    }
+
+    OfflineTilePyramidRegionDefinition regionDefinition = (OfflineTilePyramidRegionDefinition) definition;
+    setStyleUrl(regionDefinition.getStyleURL());
+    CameraPosition cameraPosition = new CameraPosition.Builder()
+      .target(regionDefinition.getBounds().getCenter())
+      .zoom(regionDefinition.getMinZoom())
+      .build();
+
+    if (!isMapInitialized()) {
+      mapboxMapOptions.camera(cameraPosition);
+      mapboxMapOptions.minZoomPreference(regionDefinition.getMinZoom());
+      mapboxMapOptions.maxZoomPreference(regionDefinition.getMaxZoom());
+      return;
+    }
+    nativeMapView.jumpTo(cameraPosition.bearing, cameraPosition.target, cameraPosition.tilt, cameraPosition.zoom);
+    nativeMapView.setMinZoom(regionDefinition.getMinZoom());
```

Comment:
Could you rather use `MapView` methods for this operations? That'd make a transition to the `ViewModel` later on easier.